### PR TITLE
Some command block gui and others mappings 

### PR DESCRIPTION
**Pull request by marcin212**

### CODE REVIEW

File: mappings/coi.mapping [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CLASS coi
+	FIELD a minecraftGame Lclx;
```
Comment (asiekierka) [inline mappings/coi.mapping]: Why not just "game"?

File: mappings/net/minecraft/block/entity/BlockEntityEnderChest.mapping [hunk 2]
```diff
@@ -1,7 +1,10 @@
 CLASS bmd net/minecraft/block/entity/BlockEntityEnderChest
+	FIELD a animationProgress F
+	FIELD e lastAnimationProgress F
 	FIELD f viewers I
 	FIELD g ticks I
 	METHOD Z_ tick ()V
+	METHOD a animationProgress (F)F
```
Comment (asiekierka) [inline mappings/net/minecraft/block/entity/BlockEntityEnderChest.mapping]: getAnimationProgress if anything

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/gui/CommandBlockGui.mapping [hunk 3]
```diff
@@ -8,6 +8,12 @@ CLASS crc
 	CLASS crc$3
 		METHOD a onPressed (DD)V
 			ARG 0 mouseX
+	FIELD A typeButton Lcmu;
+	FIELD B conditionalModeButton Lcmu;
+	FIELD C redstoneTriggerButton Lcmu;
+	FIELD D type Lblw$a;
+	FIELD E isConditional Z
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/CommandBlockGui.mapping]: conditional/alwaysActive for fields; isConditional/isAlwaysActive for methods. Unless there's a reason not to do it this way.

File: mappings/net/minecraft/client/gui/SimpleCommandBlockGui.mapping [hunk 4]
```diff
@@ -8,6 +8,11 @@ CLASS cqw
 	CLASS cqw$3
 		METHOD a onPressed (DD)V
 			ARG 0 mouseX
+	FIELD a consoleCommandTextField Lcmy;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/SimpleCommandBlockGui.mapping:11]: Do we need the TextField/Button suffixes everywhere? It's a Gui class, most of the fields will be Gui elements... Something to think about, not necessarily correct.

**Review by jamierocks — APPROVED** [review]
Looks good :+1:

**Pull request merged**